.travis.yml

```diff
@@ -19,7 +20,7 @@ script:
 
   - echo 'test webgl-stub' && echo -en 'travis_fold:start:script.test\\r'
   - npm run build
-  - npm run test -- --browsers ChromeCI --webgl-stub --failTaskOnError --suppressPassed
+  - npm run test -- --browsers FirefoxHeadless --webgl-stub --failTaskOnError --suppressPassed
```

Comment:
One of these should still be `ChromeCI` right?

Reply:
Whoopes, yes I'll fix it.  Now that I think about it, probably better to have Firefox run on the unbuilt version since almost all Cesium devs use Chrome locally.